Sources/Optimizely/OptimizelyClient.swift

```diff
@@ -744,7 +750,7 @@ open class OptimizelyClient: NSObject {
             throw OptimizelyError.eventKeyInvalid(eventKey)
         }
         
-        sendConversionEvent(eventKey: eventKey, userId: userId, attributes: attributes, eventTags: eventTags)
+        sendConversionEvent(eventKey: eventKey, userId: userId, attributes: attributes, eventTags: eventTags, clientName: self.clientName)
```

Comment:
I think we can change `Utils.sdkVersion` and `Utils.swiftSdkClientName` to update when customize. In that way, we do not change these apis.